csharp/ql/test/library-tests/csharp9/LocalFunctionAttribute.ql

```diff
@@ -0,0 +1,4 @@
+import csharp
+
+from LocalFunction lf
+select lf, lf.getAnAttribute()
```

Comment:
I suggest moving this test into `LocalFunctions.ql` (see https://github.com/github/codeql/pull/4612/files/bd8c4a85ffe5f0830242ac3c3a0de3e68ad4d8ad#r529379720 ).